File: src/app/models/vehiculo/vehiculo.ts
//zona vehiculo
export interface Vehiculo{
  id_chasis_vehiculo:number;
  patente_vehiculo:string;
  imagen_vehiculo:string;
  color_vehiculo:string;
  cilindrada_motor:number;
  tipo_combustible:string;
  categoria_vehiculo:string;
  precio_vehiculo:number;
  peso_vehiculo:number;
}

//zona de marca
export interface Marca{
  id_marca:string;
  logo_marca:string;
}

File: src/main/java/com/loquendodev/carhub/models/entity/Vehiculo.java
package com.loquendodev.carhub.models.entity;

import jakarta.persistence.Entity;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@AllArgsConstructor
@NoArgsConstructor
@Data
@Entity
public class Vehiculo {

    private Integer id_vehiculo;
    private String id_chasis_vehiculo;
    private String patente_vehiculo;
    private String imagen_vehiculo;
    private String color_vehiculo;
    private Double cilindrada_motor;
    private String tipo_combustible;
    private String categoria_vehiculo;
    private Double precio_vehiculo;
    private Double peso_vehiculo;


}
